Add Stream IoT-X instructions

janjongboom opened the pull request:
@AnotherButler @dironsARM I believe Irit is OOO, could either of you go over this? We need this published before MWC next week.

janjongboom committed:
Separate LORIOT instructions from intro-to-lora.md

janjongboom committed:
Instructions for IoT-X

janjongboom committed:
It's -> its

janjongboom committed:
Fix mDot example for IoT-X

janjongboom committed:
Merge branch 'iotx' of github.com:ARMmbed/lora-docs into iotx

AnotherButler commented:
@janjongboom Sure thing

Commit:
Copy edit LoRa docs

Edit for active voice and make other minor grammar edits

AnotherButler commented on docs/intro-to-lora.md:
```diff
@@ -1,81 +1,79 @@
-* [Kerlink IoT station](http://www.kerlink.fr/en/products/lora-iot-station-2/lora-iot-station-868-mhz). Expensive (around 1,200 euros), but great build quality and range.
-* [Multitech Conduit](http://www.multitech.com/brands/multiconnect-conduit). About a third of the price of the Kerlink (around 450 euros), and pretty great for small setups (put a bigger antenna on it though). Multitech also has a [rugged outdoor](http://www.multitech.com/brands/multiconnect-conduit-ip67) version.
-* Building your own with a Raspberry Pi and an [IMST iC880A](http://webshop.imst.de/catalogsearch/result/?q=iC880A) concentrator. At around 230 euros, this is the most cost-efficient option.
+* [Kerlink IoT station](http://www.kerlink.fr/en/products/lora-iot-station-2/lora-iot-station-868-mhz). Expensive (around 1,200 euros) but great build quality and range.
```
I can't get the Kerlink IoT station link to work. Could you please confirm it is correct?

AnotherButler commented:
Nice work. Please see my suggested edits, and the one comment I left on them.

Commit:
Edit LoRa docs

Pull in changes from PRs #9 and #10
-add note on LoRa vs. LoRaWAN
-fix bullets
-add explanation about spread factor in LoRaWAN

AnotherButler commented:
I added a commit pulling in edits from PRs #9 and #10. If you'd like me to undo that and make it separate PRs, please let me know.
